Formula/kubernetes-cli.rb

```diff
@@ -36,7 +36,11 @@ def install
 
       # Install zsh completion
       output = Utils.popen_read("#{bin}/kubectl completion zsh")
-      (zsh_completion/"_kubectl").write output
+      (zsh_completion/"_kubectl").write <<-EOS.undent
+        #compdef kubectl
+        #{output}
+        _complete kubectl 2> /dev/null
```

Comment:
Right you are @cblecker. It was kept to avoid the "`... can only be called from completion function`" message if sourced directly. But since we are not doing that, it can go away.

Update pushed.